CHANGELOG.md

```diff
@@ -67,6 +67,9 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed NullPointerException when trying to set a special field or mark an entry through the menu without having an open database
 - Fixed [#1257](https://github.com/JabRef/jabref/issues/1324): Preferences for the BibTeX key generator set in a version prior to 3.2 are now migrated automatically to the new version
 - Fixed [#1716](https://github.com/JabRef/jabref/issues/1716): `@`-Symbols stored in BibTeX fields no longer break the database
+- Fixed [#1499](https://github.com/JabRef/jabref/issues/1499): {} braces are now treated correctly in in author/editor
+- Fixed [#1531](https://github.com/JabRef/jabref/issues/1531): \relax can be used for abbreviation of author names
```

Comment:
No, that is both related and relevant. ;-)

I'll fix it tomorrow (later today).

Den 17 aug. 2016 00:08 skrev "Christoph" <notifications@github.com>:

> In CHANGELOG.md
> <https://github.com/JabRef/jabref/pull/1755#discussion_r75030057>:
>
> > @@ -67,6 +67,9 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
> >  - Fixed NullPointerException when trying to set a special field or mark an entry through the menu without having an open database
> >  - Fixed [#1257](https://github.com/JabRef/jabref/issues/1324): Preferences for the BibTeX key generator set in a version prior to 3.2 are now migrated automatically to the new version
> >  - Fixed [#1716](https://github.com/JabRef/jabref/issues/1716): `@`-Symbols stored in BibTeX fields no longer break the database
> > +- Fixed [#1499](https://github.com/JabRef/jabref/issues/1499): {} braces are now treated correctly in in author/editor
> > +- Fixed [#1531](https://github.com/JabRef/jabref/issues/1531): \relax can be used for abbreviation of author names
>
> I would put the \relax in the code-quotes, but that is just a small
> unrelated thing..;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JabRef/jabref/pull/1755/files/752c54d5d96fbfd2861970a8fb86463f0ac1ad86#r75030057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHvRQedYCQBoTKdLOEle7MnEtbznpt5cks5qgjTZgaJpZM4Jl1UE>
> .
>